styles/style.css

```
@@ -0,0 +1,65 @@
+header{
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+.page-title {
```

This can directly target the `h1` itself:

```suggestion
h1 {
```